runtime/gc_base/StackSlotValidator.cpp

```diff
@@ -100,8 +112,31 @@ MM_StackSlotValidator::reportStackSlot(MM_EnvironmentBase* env, const char* mess
 	j9tty_printf(PORTLIB, "%p:\tstack=%p-%p\n", walkThread, walkThread->stackObject + 1, walkThread->stackObject->end);
 	Trc_MM_StackSlotValidator_stack(env->getLanguageVMThread(), walkThread->stackObject + 1, walkThread->stackObject->end);
 
-	releaseOMRVMThreadName(walkThread->omrVMThread);
-	
+	if (NULL != walkThread->omrVMThread) {
+		releaseOMRVMThreadName(walkThread->omrVMThread);
+	}
+#if JAVA_SPEC_VERSION >= 19
+	else if (NULL != walkThread->currentContinuation) {
+		releaseContinuationThreadName(threadName);
+	}
+#endif /* JAVA_SPEC_VERSION >= 19 */
+
 	Trc_MM_StackSlotValidator_reportStackSlot_Exit(env->getLanguageVMThread());
 }
 
+char *
+MM_StackSlotValidator::getContinuationThreadName(void *continuation)
+{
+	char *threadName = NULL;
+	threadName = malloc(64);
```

Comment:
While this is only to be used where there are problems on stack (which means VM won't run much longer beyond this), let's keep to code free for native malloc/free and use Forge API.
Probably MM_AllocationCategory::OTHER or DIAGNOSTIC.

Reply:
minor but let's pass `J9VMContinuation *`